cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -167,6 +168,35 @@ jobs:
           cat uploads.txt
           gh release upload ${{ github.ref_name }} $(cat uploads.txt)
           echo "uploaded!"
+
+{{%- if tap %}}
+  upload-homebrew-formula:
+    needs: upload-local-artifacts
+    runs-on: {{{ global_task.runner }}}
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      RELEASES: ${{ needs.upload-local-artifacts.outputs.releases }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: {{{ tap }}}
+      # So we have access to the formula
+      - name: Fetch local artifacts
+        run: |
+          gh release download ${{ github.ref_name }} --dir Formula --repo ${GITHUB_REPOSITORY}
```

Comment:
Is there a reason to explicitly specify repo?

OH. I see we're running the CI in the target repo, that makes sense.